### PR TITLE
Add supporting independent expenditures for candidates

### PR DESCRIPTION
**Pull request by tdooner**
[Closes caciviclab/disclosure-backend-static#95]

In that issue, Suzanne noticed that we feature "Expenditures Opposing
Candidate" on the candidate's detail page, however we don't include
Expenditures supporting candidate.

Not many candidates have independent expenditures, but those that do --
Huber Trenado, Jumoke Hodge, and James Harris (three candidates for the
school board) -- have quite a lot (more than $100k each).

This also makes the section conditionally hidden if there are no
independent expenditures for/against the candidate. I'm not confident
this is always the right move (it might be nice for a user to know that
a candidate is receiving no IE money) but it seems like a reasonable
first step since most candidates have zero IE money.

### CODE REVIEW

File: _candidates/dan-kalb.md [hunk 1]
```diff
@@ -92,9 +93,7 @@ expenditures_by_type:
   'Information Technology Costs (Internet, E-mail)': 4226.01
   Independent Expenditure Supporting/Opposing Others: 400
   Transfer Between Committees of the Same Candidate/sponsor: 13500
-ballots:
-- _ballots/oakland/2016-11-08.md
```
Comment (tdooner) [inline _candidates/dan-kalb.md]: Should this have been removed? This seems wrong. I only ran `gulp pull`.

File: _candidates/jeff-sheehy.md [hunk 2]
```diff
@@ -1,39 +1,71 @@
 ---
-id: 28
+id: 47
 name: Jeff Sheehy
 photo_url: null
-website_url: "http://jeffsheehy.org/"
+website_url: null
```
Comment (tdooner) [inline _candidates/jeff-sheehy.md]: He has a row in the candidates spreadsheet (on the "All Candidates") tab now, that can be used to populate info like this.

Comment (tdooner) [inline _candidates/jeff-sheehy.md]: This data can be automatically pulled in by the API now by modifying Sheehy's row in the "All Candidates" tab of the candidates spreadsheet.

Comment (tdooner) [reply _candidates/jeff-sheehy.md]: (I've added the website here, so it should come back next time we sync the data.)

File: _candidates/dan-kalb.md (hunk 1 above)
Comment (adborden) [reply _candidates/dan-kalb.md]: Right, you don't want to remove this. The gulp commands are dumb, they don't merge with the existing files, so you have to manually check for these and exclude them from the diff. I use `git add -p` for this.

File: _candidates/ken-houston.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 first_name: Ken
 last_name: Houston
-ballot_item: 9
-office_election: 9
+ballot_item: 10
+office_election: 10
 bio: null
-committee_name: null
-is_accepted_expenditure_ceiling: null
+committee_name: No committee registered
```
Comment (adborden) [inline _candidates/ken-houston.md]: Can we just remove this from the spreadsheet? It's better if it shows up as `null`. Also, I'm not sure we're using this? We can just grab the committee from the filer id.

File: _referendum_opposing/oakland/2016-11-08/d.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (adborden) [inline _referendum_opposing/oakland/2016-11-08/d.md]: You can remove this file. The gulp task just shoves all the referendums into the Nov 2016 election, but this is actually on the June '18 ballot. I think I'm working on a fix on another branch and will add them in the right place in that PR.

This is also the case for `tbd1` and `tbd2`, which should be on the Nov '18 ballot. All of these will all appear under `_referendums`, `_referendum_opposing`, and `_referendum_supporting`. So, 9 files in total to be removed... sorry :grimacing:

Comment (adborden) [reply _referendum_opposing/oakland/2016-11-08/d.md]: Ah, I added election guessing logic in #62

**Comment by adborden** [comment]
I'm going to rebase this branch and get this merged. Now that the finance data is updated with what you need, `npm run pull` isn't needed anymore.

**Comment by adborden** [comment]
I rebased the changes into `tdooner-add-ie-supporting` in the PR ^. It felt wrong to force push to your branch so I created a new PR instead.

**Comment by adborden** [comment]
Closing. This is set in the other PR. Thanks!

**Pull request closed without merging**